### PR TITLE
README.md:add missing step to build instructions

### PR DESCRIPTION
**Pull request by andy5995**
addresses problem mentioned in #134
[skip ci]

### CODE REVIEW

**Comment by shnupta** [comment]
I think this will be okay for now, however, we may change this in the near future. Does it have to create the `build/src` directory?

**Comment by andy5995** [comment]
Ok, this is good to go.

File: configure.ac [hunk 1]
```diff
@@ -14,7 +14,8 @@ AM_MAINTAINER_MODE
 AM_GNU_GETTEXT([external], [need-formatstring-macros])
 AM_GNU_GETTEXT_VERSION([0.19.8])
 
+# The Makefile.* filenames in po/ are different than the others, and that's intentional,
```
Comment (andy5995) [inline configure.ac]: This is an unrelated comment.

File: configure.ac [hunk 2]
```diff
@@ -14,7 +14,8 @@ AM_MAINTAINER_MODE
...
+# according the GNU Gettext documentation.
 AC_CONFIG_FILES([       Makefile
-                        src/Makefile
```
Comment (andy5995) [inline configure.ac]: To have the binary build outside of src, I moved the contents of src/Makefile to the main Makefile.
